scrapy/core/downloader/__init__.py

```diff
@@ -60,8 +60,8 @@ def _get_concurrency_delay(concurrency, spider, settings):
     if hasattr(spider, 'download_delay'):
         delay = spider.download_delay
 
-    if hasattr(spider, 'max_concurrent_requests'):
-        concurrency = spider.max_concurrent_requests
+    if hasattr(spider, 'slot_concurrent_requests'):
+        concurrency = spider.slot_concurrent_requests
```

Comment:
This is not a new line being added rather just a renaming happening. It might not already be covered by the test cases that is why the check does not succeed and shows,
```
Codecov / codecov/patch

scrapy/core/downloader/__init__.py#L64

Added line #L64 was not covered by tests
```